### ISSUE TITLE
Is there a way to set this up using RNMP instead?

### ISSUE THREAD

**Issue by quadsurf** [alinz/react-native-webview-bridge#166]

**Comment by alinz** [comment]
@quadsurf are you talking about react-native package manager (RNPM)? if that you are talking about then version 1 is compatible but version 2 is not at the moment.

**Comment by quadsurf** [comment]
@alinz yes, referring to the RN package manager. Once v2 has it, would it need to be re-added to Core as a new/separate PR?
